app/src/main/res/layout/layout_filter_reveal.xml

```diff
@@ -46,22 +47,50 @@
             android:paddingRight="15dp"
             android:orientation="vertical">
 
-           <TextView
-               android:layout_width="wrap_content"
-               android:layout_height="wrap_content"
-               android:text="Select Language"
-               android:textSize="18sp"
-               android:textColor="@android:color/white"/>
-
-            <TextView
-                android:layout_width="wrap_content"
+            <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:id="@+id/tv_language"
-                android:textColor="@android:color/white"
-                android:drawableRight="@drawable/ic_arrow_drop_down"
-                android:textSize="18sp"
-                android:drawablePadding="10dp"
-                android:layout_marginTop="10dp"/>
+                android:gravity="bottom|end"
+                android:orientation="vertical">
+
+               <TextView
+                   android:layout_width="wrap_content"
+                   android:layout_height="wrap_content"
+                   android:text="Select Language"
```

Comment:
Please use @strings here to be future proof for #9

Reply:
When developing I actively decided against that. I saw there was a PR open that was adressing that, but I decided to stick with the current way Strings are embedded into the app (namely hardcoded).

I did this for 2 reasons:
a) In case he does not want to use string ressources (unlikely)
b) So when he merges this branch with the master branch it will be consistent.

I would rather use string ressources aswell. If #9 gets integrated, I will update these values with string ressource values.

Reply:
Seems like a fair plan, you can always rebase against master later!